automation/publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -xe
+
+# This script publishes macvtap-cni by default at quay.io/kubevirt
+# organization.
+# To publish elsewhere export the following env vars
+# IMAGE_REGISTRY
+# IMAGE_REPO
+# To run it just do proper docker login and automation/publish.sh
+
+source automation/check-patch.setup.sh
+cd ${TMP_PROJECT_PATH}
+make \
+    IMAGE_REGISTRY=${IMAGE_REGISTRY:-quay.io}  \
```

Comment:
No need to, it is already in Makefile defaults

Reply:
Done.